nucliadb_node/src/http_server/mod.rs

```diff
@@ -33,13 +34,13 @@ pub struct MetricsServerOptions {
     pub default_http_port: u16,
 }
 
-pub async fn run_http_metrics_server(options: MetricsServerOptions) {
+pub async fn run_http_server(options: MetricsServerOptions) {
```

Comment:
NIT: you could also rename `MetricsServerOptions` to `ServerOptions` for example, as the function is now more generic